src/components/BoxApproval/BoxApproval.jsx

```diff
@@ -1,38 +1,154 @@
-      <h1 className="boxApprovalHeading">Box Approval</h1>
-      <div className="boxApprovalTabs">
-        <Button
-          onClick={() => setCurrentTab('underReview')}
-          width="50%"
-          className={`button-${currentTab === 'underReview'}`}
-        >
-          <p className="buttonText">Under Review</p>
-        </Button>
-        <Button
-          onClick={() => setCurrentTab('previouslyReviewed')}
-          width="50%"
-          className={`button-${currentTab === 'previouslyReviewed'}`}
-        >
-          <p className="buttonText">Previously Reviewed</p>
-        </Button>
+    <ChakraProvider>
+      <div className={styles['box-approval']}>
+        <Tabs align="center" variant="line">
+          <div>
+            <TabList>
+              <Tab onClick={() => loadBoxesUnderStatus('under review')}>Under Review</Tab>
+              <Tab onClick={() => loadBoxesUnderStatus('pending changes')}>Pending Changes</Tab>
+              <Tab onClick={() => loadBoxesUnderStatus('evaluated')}>Evaluated</Tab>
+            </TabList>
+          </div>
+          <div className={styles['box-list']}>
+            <TabPanels>
+              {/* 'Under Review' section */}
+              <TabPanel>
+                <div>
+                  {relocationBoxesUnderReview.map(boxData => mapDataToRelocationBox(boxData))}
+                </div>
+                <div>{pickupBoxesUnderReview.map(boxData => mapDataToPickupBox(boxData))}</div>
+              </TabPanel>
+              {/* 'Pending Changes' section */}
+              <TabPanel>
+                <div>{relocationBoxesPending.map(boxData => mapDataToRelocationBox(boxData))} </div>
+              </TabPanel>
+              {/* 'Evaluated' section */}
+              <TabPanel>
+                <div>
+                  {relocationBoxesEvaluated.map(boxData => mapDataToRelocationBox(boxData))}
+                </div>
+                <div>
+                  {pickupBoxesEvaluated.map(boxData => mapDataToPickupBox(boxData))}
+                  <RejectBoxPopup />
+                  <RequestChangesPopup />
+                </div>
+              </TabPanel>
+            </TabPanels>
+          </div>
+        </Tabs>
+        <div className={styles.legend}>
+          <div className={styles['request-changes-row']}>
+            <img className={styles['request-changes-icon']} src={RequestChangesMiniIcon} alt="" />
+            <p className={styles['request-changes-text']}>Request Changes</p>
+          </div>
+          <div className={styles['relocate-box-row']}>
+            <img className={styles['relocate-box-icon']} src={RelocateBoxIcon} alt="" />
+            <p className={styles['relocate-box-text']}>Relocated</p>
+          </div>
+          <div className={styles['pickup-box-row']}>
+            <img className={styles['pickup-box-icon']} src={PickupBoxIcon} alt="" />
+            <p className={styles['pickup-box-text']}>Picked Up</p>
+          </div>
+        </div>
```

Comment:
Is it possible to make this a part of the scrollable section? If I zoom in too much, then the legend cuts off and I'm not able to scroll down to see all the options.

Reply:
Looks good, thanks!